app/controllers/spotlight/appearances_controller.rb

```diff
@@ -22,20 +27,34 @@ def edit
 
     protected
 
+    def update_masthead
+      @exhibit.masthead ||= Spotlight::Masthead.new
+      @exhibit.masthead.update(params.require(:exhibit).require(:masthead_attributes).permit(featured_image_params))
+      @exhibit.save
```

Comment:
`masthead = @exhibit.masthead || @exhibit.build_masthead` might be a little better